4-storage/relational-database-shredder/src/main/scala/com.snowplowanalytics.snowplow.storage/hadoop/utils/JsonUtils.scala

```diff
@@ -12,7 +12,7 @@
  */
 package com.snowplowanalytics
 package snowplow
-package enrich
+package storage
 package hadoop
```

Comment:
I left the old code in place, this is an artifact from the change of package #3036